src/main/java/uk/gov/hmcts/reform/slc/services/steps/sftpupload/FtpClient.java

```diff
@@ -94,6 +94,17 @@ public void upload(PdfDoc pdfDoc) {
         });
     }
 
+    public void deleteReport(String reportPath) {
+        runWith(sftp -> {
+            try {
+                sftp.rm(reportPath);
+                return null;
```

Comment:
does it need a return statement if the block is void on it's own?

Reply:
yup, `runWith` takes `Function<SFTPClient, T>`.
it would have to be `Consumer<SFTPClient>`.

Reply:
aye